### PR TITLE
doc: add minutes for meeting 27 Nov 2019

### PR DESCRIPTION
**Pull request by mhdawson**

### CODE REVIEW

File: meetings/2019-11-27.md [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+
+
+### nodejs/admin
+
+* Collaborator Summit - Montreal 2019 [#416](https://github.com/nodejs/admin/issues/416)
+  * Reminder to submit sessions, sooner the better as schedule probably filling up.
+
+## Strategic Initiatives
+
+* N-API team working on Workshop for Node +JS Interactive hope to see you there.
```
Comment (cjihrig) [inline meetings/2019-11-27.md]:
```suggestion
* N-API team working on Workshop for Node+JS Interactive hope to see you there.
```

File: meetings/2019-11-27.md [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+
+
+* Anna Henningsen @addaleax (TSC)
+* Colin Ihrig @cjihrig (TSC)
+* Jeremiah Senkpiel @Fishrock123 (TSC)
+* Joyee Cheung @joyeecheung (TSC)
+* Michael Dawson @mhdawson (TSC)
+* Rich Trott @Trott (TSC)
+* Sam Roberts @sam-github (TSC)
+
```
Comment (Trott) [inline meetings/2019-11-27.md]: I believe @tniessesn was there for most of the meeting.

File: meetings/2019-11-27.md [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+* Streams - moving repo into core, planned session at Node+JS Interactive
+  * good likelihood will not move forward unless we find new champion.
+
+* Open Web standards - defer to discussion when Myles is present.
+
+* Python 3 and Gyp, no update.
+
+* Startup Performance 
+  * People who used shared library are having issues with code cache, and snapshot builder.
+    currently fixing build process to address
```
Comment (cjihrig) [inline meetings/2019-11-27.md]:
```suggestion
    Currently fixing build process to address.
```

File: meetings/2019-11-27.md [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+  * also working to fix build process that is currently 2 steps, one without unresolved symbols
+
+* Build Resources
+  * Michael gave presentation to board as request, back on the board side for next comeback.
```
Comment (cjihrig) [inline meetings/2019-11-27.md]:
```suggestion
  * Michael gave presentation to board as requested, back on the board side for next comeback.
```

File: meetings/2019-11-27.md (hunk 2 above)
Comment (Trott) [reply meetings/2019-11-27.md]: Er, I mean @tniessen ....

Comment (mhdawson) [reply meetings/2019-11-27.md]:
``` suggestion
* Tobias Nießen @tniessen (TSC)
````

File: meetings/2019-11-27.md [hunk 5]
```diff
@@ -0,0 +1,90 @@
...
+
+
+* Anna Henningsen @addaleax (TSC)
+* Colin Ihrig @cjihrig (TSC)
+* Jeremiah Senkpiel @Fishrock123 (TSC)
+* Joyee Cheung @joyeecheung (TSC)
+* Michael Dawson @mhdawson (TSC)
+* Rich Trott @Trott (TSC)
+* Sam Roberts @sam-github (TSC)
+* Tobias Nießen @tniessen (TSC)
```
Comment (mhdawson) [inline meetings/2019-11-27.md]:
``` suggestion
* Tobias Nießen @tniessen (TSC)

```

**Pull request merged**